lib/cb/models/implementations/saved_search.rb

```diff
@@ -36,18 +36,18 @@ def initialize(args = {})
         @search_parameters          = SearchParameters.new(args['SavedSearchParameters'] || {})
       end
 
-      def create_to_xml
-        <<-eos
-          <Request>
-            <HostSite>#{host_site}</HostSite>
-            <Cobrand>#{cobrand}</Cobrand>
-            <SearchName>#{search_name}</SearchName>
-            #{search_parameters.to_xml}
-            <IsDailyEmail>#{is_daily_email.to_s.upcase}</IsDailyEmail>
-            <ExternalUserID>#{external_user_id}</ExternalUserID>
-            <DeveloperKey>#{Cb.configuration.dev_key}</DeveloperKey>
-          </Request>
-        eos
+      def create_to_json
+        hash = {
+          'SiteID' => site_id,
+          'Cobrand' => cobrand,
+          'EmailDeliveryDay' => email_delivery_day,
```

Comment:
You know, this brings up a case I hadn't thought of. On create, there is no actual option for email frequency. It's auto set to weekly. So, there is no use for either of these nodes (just tested that it works without them). Will remove node and check back in.